website/source/docs/drivers/rkt.html.md

```diff
@@ -33,10 +33,9 @@ The `rkt` driver supports the following configuration in the job spec:
 
 ## Task Directories
 
-The `rkt` driver does not currently support mounting the `alloc/` and `local/`
-directory. It is currently blocked by this [rkt
-issue](https://github.com/coreos/rkt/issues/761). As such the coresponding
-[environment variables](/docs/jobspec/environment.html#task_dir) are not set.
+The `rkt` driver requires support for mounting `alloc/` and `local/` directories,
```

Comment:
Ah, that makes sense. Thought it was just a documentation issue but the
driver doesn't support mounting yet either. I'll close this and wait till
there's a PR to supportf file mounting for rkt.

The link https://github.com/coreos/rkt/issues/761 indicates that it's
closed and fixed in version 0.10 of rkt.

--------------------------------------------
Charlie Drage
4096R / 0x9B3B446C
http://pgp.mit.edu/pks/lookup?op=get&search=0x622CDF119B3B446C
--------------------------------------------

On Thu, Nov 26, 2015 at 4:44 PM, Alex Dadgar <notifications@github.com>
wrote:

> In website/source/docs/drivers/rkt.html.md
> <https://github.com/hashicorp/nomad/pull/508#discussion_r46007876>:
>
> > @@ -33,10 +33,9 @@ The `rkt` driver supports the following configuration in the job spec:
> >
> >  ## Task Directories
> >
> > -The `rkt` driver does not currently support mounting the `alloc/` and `local/`
> > -directory. It is currently blocked by this [rkt
> > -issue](https://github.com/coreos/rkt/issues/761). As such the coresponding
> > -[environment variables](/docs/jobspec/environment.html#task_dir) are not set.
> > +The `rkt` driver requires support for mounting `alloc/` and `local/` directories,
>
> The rkt driver currently does not support mounting ...
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/hashicorp/nomad/pull/508/files#r46007876>.
>